emcc.py

```diff
@@ -1815,6 +1815,7 @@ def default_setting(name, new_default):
     # helping people see when they should have enabled EXIT_RUNTIME)
     if settings.EXIT_RUNTIME or settings.ASSERTIONS:
       settings.EXPORTED_FUNCTIONS += ['_fflush']
+    settings.EXPORTED_FUNCTIONS += ['_memalign']
```

Comment:
unintended? this sounds quite heavyweight otherwise, it will end up adding to a lot of programs.

Reply:
I can instead make mmapAlloc fail at runtime if the export is missing?  (like makeMalloc does?) ,  but this is more technically correct, the FS library does transitively depend on this.   

I guess I was assuming programs that include FILESYSTEM are already quite bloated/large (at least their JS side will be, no?)

Reply:
Shouldn't the deps_info change suffice for this? It's not that all FILESYSTEM programs depend on mmapAlloc, just ones that do mmap.

FILESYSTEM using code does get some bloat, that is true, but I don't think we need to increase that here unless I am missing something.

Reply:
In theory yes, in practice the JS code for the file system is monolithic, including FS, also includes all the mmap implementations.   These may well be unused at runtime, but closure will see the call to memalign and complain.      I can use the same trick we do in `makeMalloc` though I think..

Reply:
Fixed.